src/pages/index.astro

```diff
@@ -473,9 +473,97 @@ import logo from '../assets/logo.png';
+            <span>Hva skjer hvis det er tekniske problemer under sesongen?</span>
+            <svg class="w-5 h-5 text-gray-500 transition-transform group-open:rotate-180" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7"></path>
+            </svg>
+          </summary>
+          <div class="px-6 py-4 text-gray-600 border-t border-gray-100">
+            <p>
+              Vi har 99.9% oppetid og holder systemet aktivt overvåket. Skulle det oppstå problemer, har vi support tilgjengelig og backup-rutiner for å sikre at data ikke går tapt. All rapportering er også tilgjengelig offline i appen.
+            </p>
+          </div>
+        </details>
+
+        <!-- FAQ 11 -->
+        <details class="bg-white rounded-lg shadow-sm border border-gray-200 overflow-hidden group">
+          <summary class="cursor-pointer px-6 py-5 font-semibold text-lg text-gray-900 hover:bg-gray-50 transition-colors list-none flex justify-between items-center">
+            <span>Hvordan får jeg turistene til å faktisk bruke appen?</span>
+            <svg class="w-5 h-5 text-gray-500 transition-transform group-open:rotate-180" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7"></path>
+            </svg>
+          </summary>
+          <div class="px-6 py-4 text-gray-600 border-t border-gray-100">
+            <p>
+              Turistene motiveres av at de får offisiell dokumentasjon for lovlig eksport av fisk. Appen er på deres språk (engelsk/tysk), enkel å bruke, og krever ingen registrering eller personlige opplysninger. De fleste turistene ser verdien umiddelbart.
+            </p>
+          </div>
+        </details>
+
+        <!-- FAQ 12 -->
+        <details class="bg-white rounded-lg shadow-sm border border-gray-200 overflow-hidden group">
+          <summary class="cursor-pointer px-6 py-5 font-semibold text-lg text-gray-900 hover:bg-gray-50 transition-colors list-none flex justify-between items-center">
+            <span>Må turistene ha en norsk telefon eller e-postadresse?</span>
+            <svg class="w-5 h-5 text-gray-500 transition-transform group-open:rotate-180" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7"></path>
+            </svg>
+          </summary>
+          <div class="px-6 py-4 text-gray-600 border-t border-gray-100">
+            <p>
+              Nei, appen fungerer med alle telefonnumre og e-postadresser internasjonalt. Turistene trenger kun internettforbindelse for å åpne appen og registrere fangst. Ingen norsk SIM-kort eller bankkonto nødvendig.
```

Comment:
Applikasjonen bruker ikke telefonnummer, men man trenger en epost addresse. Ellers trenger man en mobil, nettbrett eller pc som man kan bruke på internett for å registrere.